app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,92 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* typography */  
+  h1 {
+    font-size: 3em;
+    margin-bottom: 1em;
+    text-align: center;
+    color: #544a47;
+  }
```

typographyのCSS(h1, h2, p, a)インデントが2つ下がってるので、特に意味がなければインデント無くしましょう 🙆 
```suggestion
h1 {
  font-size: 3em;
  margin-bottom: 1em;
  text-align: center;
  color: #544a47;
}
```